test/stdlib/TestUserInfo.swift

```diff
@@ -20,15 +20,29 @@ import StdlibUnittest
 class TestUserInfoSuper : NSObject { }
 #endif
 
-struct SubStruct {
+struct SubStruct: Equatable {
     var i: Int
     var str: String
+
```

Comment:
OK.  Just please try to spell that kind of thing out in commit messages and/or comments